src/script/util/DataDog.ts

```diff
@@ -46,15 +54,14 @@ export async function initializeDataDog(config: Configuration, domain?: string)
     version: config.VERSION,
     sessionSampleRate: 100,
     sessionReplaySampleRate: 20,
+    silentMultipleInit: true,
```

Comment:
Let's remove this one, cause we would like to be warned in case we init it twice (this would be a mistake)
```suggestion
    silentMultipleInit: true,
```